### ISSUE TITLE
Unable to generate accessibility reports for dynamic web pages

### ISSUE THREAD

**Issue by supasani** [nikulkarni/webdriver-accessibility#6]
Hi,

This tools is really awesome and works perfectly fine for static web pages. But i am facing difficulty in generating reports for dynamic web pages. 

What would you recommend? 
Is there any update coming sometime soon which is related to the above problem?

Thanks,
Sameer

**Comment by nikulkarni** [comment]
Hi Sameer, Can you explain what you mean by dynamic pages? Do you have examples? This tool is supposed to work with all sorts of web pages
